package.json

```diff
@@ -1,3 +1,4 @@
+
 {
   "name": "orionuo",
   "version": "1.0.0",
```

Comment:
version bump? :)

Reply:
na 1.1.3 ? Aspon ak ideme podla changelogu

Reply:
podľa semver by to malo ísť na 1.1.3, áno :)

Reply:
zmenene na 1.1.3